src/lib/kernel/Linux.cc

```diff
@@ -15,6 +15,11 @@
 #include <algorithm>
 #include <cassert>
 #include <cstring>
+#if defined(__GNUC__) && !defined(__clang__) && __GNUC__ < 8
```

Comment:
Did we end up using `<filesystem>`?

Reply:
Removed, along with some other unused includes